SwrveSDK/src/google/java/com/swrve/sdk/ISwrve.java

```diff
@@ -18,5 +18,7 @@
 
     void processIntent(Intent intent);
 
+    void setRegistrationId(String registrationId);
```

Comment:
@Sergio-Mira We should also add this to SwrveSDK so it can be called like this SwrveSDK.setRegistrationId